src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -241,7 +241,8 @@ class parser {
                 throw as_exception(error_collector.error());
             }
         }
-        _fdp.set_name(schema.sub()());
+        const auto& sub = schema.sub()();
+        _fdp.set_name(sub.data(), sub.size());
```

Comment:
std::string_view also works here in bazel, but protobuf used in cmake oss build on fc38 doesn't seem to have this overload.

Reply:
Oh, the vtools build seems this is enabled:
```c++
#ifdef ABSL_USES_STD_STRING_VIEW

#include <string_view>  // IWYU pragma: export

namespace absl {
ABSL_NAMESPACE_BEGIN
using string_view = std::string_view;
ABSL_NAMESPACE_END
}  // namespace absl
```

Reply:
Ah, good spot. tracking here it's gonna take a bi tof investigation to figure out how to get abseil config chagnees to propogate through its depdency rather than doing something like setting this define project wide. not sure how that's done with bzlmod yet. https://redpandadata.atlassian.net/browse/CORE-6800

Reply:
Using `std::string_view` for `absl::string_view` is enabled for the bazel build too?

https://github.com/abseil/abseil-cpp/blob/08850701d27544b4af6501154791d3d45e28b0f8/absl/base/config.h#L567

The issue is 24.4 is an old version of proto and doesn't have a string view overload. It doesn't have anything to do with absl?